tests/test_animation.py

```diff
@@ -49,7 +49,7 @@ async def animation(bot, chat_id):
         ).animation
 
 
-class TestAnimation:
+class Space:
```

Comment:
what about `AnimationSpace` like I proposed in the issue?

Reply:
"Space" alone is not sufficient to get the meaning across. With "Base" it's clear that it's a base class and my favorite in the context of https://github.com/python-telegram-bot/python-telegram-bot/pull/3426#discussion_r1051648571. "NameSpace" somewhat gets across the meaning although I like it less today than I did yesterday because we're not really defining any (test) functionality in that "namespace". "Constants" could also work.

Reply:
hmm, I like "Constants" more compared to others. Just "Constants" or something like "AnimationConstants"?

Reply:
a `Test<ClassName>`, `_Test<ClassName>` or at very last `<ClassName>` prefix is a must for me. since you agree on subclassing, I definitely vote for `Base` though. That way we also don't get weird naming if at some point we e.g. start adding auxiliary functions to that class.